Python/libs/RemoteApi.py

```diff
@@ -1,6 +1,6 @@
 from typing import Dict, Any
 
-from HttpCommunicationUtil import HttpCommunicationUtil
+from .HttpCommunicationUtil import HttpCommunicationUtil
```

Comment:
Weird that the dot is required. Is that a path issue from where the library is installed?

Reply:
Relative import from the same directory as the current module, otherwise python starts looking top level and gets lost